### ISSUE TITLE
File:// protocol doesn't work - "Link address is not valid."

### ISSUE THREAD

**Issue by kine** [MicrosoftDocs/dynamics365smb-devitpro-pb#1891]
I am not sure if the documentation is wrong or it is a bug, but if I use link in format "file://server/sharedfolder/..." and I try to open it, I will get error: 
Link address is not valid.

Is the file:// protocol supported or not?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b7543dd1-8323-2341-7eb9-d2cc9fa81083
* Version Independent ID: 04e0fad0-745e-7434-4ec8-8a1df588a098
* Content: [System.Hyperlink Method - Business Central](https://docs.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/methods-auto/system/system-hyperlink-method)
* Content Source: [dev-itpro/developer/methods-auto/system/system-hyperlink-method.md](https://github.com/MicrosoftDocs/dynamics365smb-devitpro-pb/blob/main/dev-itpro/developer/methods-auto/system/system-hyperlink-method.md)
* Service: **dynamics365-business-central**
* GitHub Login: @SusanneWindfeldPedersen
* Microsoft Alias: **solsen**

**Comment by SusanneWindfeldPedersen** [comment]
Hello, this should work and judging from your mock-up file path, I assume that you're file is stored on a network file share, and not locally. There's a note in the topic about that. If that doesn't work, I think it could be a product bug. Thanks.

**Comment by kine** [comment]
Yes, the note is why we are using the network share. But it doesn't work. 
Based on message from Marco Mels here> https://community.dynamics.com/business/f/dynamics-365-business-central-forum/410390/intranet-links-on-bc-on-prem-link-address-is-not-valid it seems that the file:// is not possible to use even for network shares. Thus I opened this issue because it seems that the documentation is not up-to-date with reality.
